### ISSUE TITLE
As a user, I will receive a confirmation e-mail.

### ISSUE THREAD

**Issue by clarkngo** [clarkngo/curriculum#20]
- [x] create a dummy e-mail account using GMail or other provider for this Rails application
- [x] add the confirmable option to the User model and create the appropriate migration file.  [Please read](https://github.com/plataformatec/devise/wiki/How-To:-Add-:confirmable-to-Users).
- [ ] create a file called `config/initializers/mail.rb` file to hold your e-mail settings for development, test, and production environments. [Please read](https://guides.rubyonrails.org/action_mailer_basics.html#action-mailer-configuration) sections 6, 7, and 8.
- [x] install and configure Figaro gem to manage environment variables for your development, test, and production environments.  [Please watch](https://www.youtube.com/watch?v=KZvNyRPMgto). 
- [ ] write a feature spec which tests that the confirmation e-mail message has been successfully sent to the user

**Comment by clarkngo** [comment]
Do i need to follow this documentation (https://edgeguides.rubyonrails.org/action_mailer_basics.html and) create the mailer first (`rails generate mailer UserMailer`)?

**Comment by conradwt** [comment]
No, I recommend using the link [here](https://guides.rubyonrails.org/) because `edgeguides` refers to Rails 6 beta at this time.  Next, you're correct that you can use `rails generate mailer UserMailer`

**Comment by clarkngo** [comment]
`rails generate mailer UserMailer`outputs:

```
create  app/mailers/user_mailer.rb
create  app/mailers/application_mailer.rb
invoke  erb
create    app/views/user_mailer
create    app/views/layouts/mailer.text.erb
create    app/views/layouts/mailer.html.erb
invoke  test_unit
create    test/mailers/user_mailer_test.rb
create    test/mailers/previews/user_mailer_preview.rb
```
I should remove the following files?
```
test/mailers/user_mailer_test.rb
test/mailers/previews/user_mailer_preview.rb
```

**Comment by conradwt** [comment]
@clarkngo There's no need to create a `UserMailer` here because this functionality exists with `Devise`.  Thus, I recommend that you review `Devise` docs.
